src/main.js

```diff
@@ -0,0 +1,15 @@
+$(function() {
+  $('form').on('submit', function (event) {
+    event.preventDefault()
+    var id = $('titleSearch').val()
```

Comment:
Right now I've got titleSearch as an id, so this could be changed to '#titleSearch" to access it. Alternatively, I can change it to a class if that's easier.